web/src/components/Navigation/BottomNav.tsx

```diff
@@ -0,0 +1,25 @@
+import { MdAccountCircle, MdAddCircle, MdHome } from "react-icons/md";
+
+import BottomNavLinkItem from "./BottomNavLinkItem";
+
+export default function BottomNav() {
+  return (
+    <nav
+      className={
+        "sticky bottom-0 -mt-12 h-12 w-screen rounded-t-xl bg-slate-300/90 sm:hidden"
+      }
+    >
+      <ul className={"z-10 flex h-full list-none items-center justify-evenly"}>
+        <BottomNavLinkItem href={"/feed"}>
```

Comment:
Does our formatting enforce braces around string literals? They aren't required by jsx and seem especially silly here.

Reply:
It doesn't enforce them, it is just force of habit of mine lol

Reply:
Fair enough; I have the opposite habit